### ISSUE TITLE
Use a client/server architecture

### ISSUE THREAD

**Issue by traverseda** [traverseda/PyCraft#6]
Exact implementation details definitely need to be discussed. But I recommend a dead-simple implementation using python 3.5's async features, which #1 allows.

**Comment by traverseda** [comment]
I'd like to separate this as far as possible, so that you could write a generic client in, say, golang. This means sending texture data over the wire, and minimizing client-side processing in general. Clients should render, and send controls.

**Comment by nitori** [comment]
Do you intend to make it python3 only? If so, I might clean up some legacy parts.

**Comment by traverseda** [comment]
Yeah, that's the intent. I don't see a clean way to do networking otherwise.

**Comment by traverseda** [comment]
If you decide not to, let me know what legacy parts stand out.

Thanks for the help either way.

**Comment by traverseda** [comment]
If you decide not to, let me know what legacy parts stand out.

Thanks for the help either way.

**Comment by traverseda** [comment]
Our core datatype is a key value store, so something like 

http://symas.com/mdb/

might let as do reasonable multi-server gameworlds.
